test/p4rt/suite.go

```diff
@@ -25,7 +25,7 @@ func (s *TestSuite) SetupTestSuite(c *input.Context) error {
 	umbrella := charts.CreateUmbrellaRelease()
 	r := umbrella.
 		Set("global.image.registry", registry).
-		Set("import.onos-cli.enabled", false). // not needed - can be enabled by adding '--set onos-umbrella.import.onos-cli.enabled=true' to helmit args for investigation
+		Set("import.onos-cli.enabled", true). // not needed - can be enabled by adding '--set onos-umbrella.import.onos-cli.enabled=true' to helmit args for investigation
```

Comment:
We don't need to enable this. You can enable via helmit command

Reply:
I left it there for ease during testing, but I can change it back now.